replies.go

```diff
@@ -32,19 +32,19 @@ var nightlies = "https://impactclient.net/ImpactInstaller.<EXT>?nightlies=true"
 var replies = []Reply{
 	{
 		pattern: `forge`,
-		message: "Use the [installer](https://impactclient.net/) to install Forge (1.12.2 only)\nBaritone 1.16 for Forge can be downloaded from [here](https://github.com/cabaletta/baritone/releases/download/v1.6.1/baritone-standalone-forge-1.6.1.jar)",
+		message: "Use the [installer](https://impactclient.net/) to install Forge (1.12.2 only)\nDownload standalone Baritone from [GitHub](https://github.com/cabaletta/baritone/releases).",
 	},
 	{
 		pattern: `faq|question`,
 		message: "[Setup/Install FAQ](https://github.com/impactdevelopment/impactclient/wiki/Setup-FAQ)\n[Usage FAQ](https://github.com/impactdevelopment/impactclient/wiki/Usage-FAQ)",
 	},
 	{
 		pattern: `defender|virus|mcafee|norton|trojan|\brat\b`,
-		message: "[Please read this thread regarding Impact being flagged by antiviruses](https://github.com/ImpactDevelopment/ImpactIssues/wiki/Setup-FAQ#my-antivirus-says-the-installer-is-a-virus-is-it-a-virus)\n\n[Direct download link after adfly](https://impactdevelopment.github.io/?brady-money-grubbing-completed=true)",
+		message: "[FAQ: My antivirus says the installer is a virus! Is it a virus?](https://github.com/ImpactDevelopment/ImpactIssues/wiki/Setup-FAQ#my-antivirus-says-the-installer-is-a-virus-is-it-a-virus)\n\n[Direct download link after AdFly](https://impactdevelopment.github.io/?brady-money-grubbing-completed=true)",
 	},
 	{
 		pattern: `tutorial|(impact|install|download).*(on|for) (windows|linux|mac)`,
-		message: "Tutorial videos for downloading and installing the client:\n[Windows](https://www.youtube.com/watch?v=QP6CN-1JYYE)\n[Mac OSX](https://www.youtube.com/watch?v=BBO0v4eq95k)\n[Linux](https://www.youtube.com/watch?v=XPLvooJeQEI)\n",
+		message: "Tutorial videos for downloading and installing the client:\n[Windows](https://www.youtube.com/watch?v=9IV_NC377pg)\n[macOS](https://www.youtube.com/watch?v=BBO0v4eq95k)\n[Linux](https://www.youtube.com/watch?v=XPLvooJeQEI)\n",
```

Comment:
New Windows video by BongEatBong looks good. But why go from `Mac OSX` to `macOS`?

Reply:
cause apple stopped calling it osx in favor of macos

Reply:
Didn't know that, thanks. Could we be consistent with capitalization though - `MacOS` or `Mac OS`?

Reply:
apple seems to call it "macOS"
![image](https://user-images.githubusercontent.com/24397680/103466337-42c6b180-4d12-11eb-8a4c-86ab763964e9.png)

Reply:
> apple seems to call it "macOS"

On that basis, maybe we should be saying `GNU/Linux` instead of just `Linux` :wink:

Reply:
@SoNothingMC do it for the funny

Reply:
aight